climt/_components/gfs/component.py

```diff
@@ -121,82 +112,36 @@ def __init__(
             damping_timescale (float, optional):
                 The damping timescale in :math:`s` to use for top-of-model Rayleigh damping.
 
-            dry_pressure (float, optional):
-                The dry pressure decides the mass of the dry atmosphere, to be used by the
-                dycore to keep the dry mass of the atmosphere constant for the duration of the run.
-                The default value corresponds to :math:`10^5\ Pa`, which is suitable for the
-                current atmospheric mass on earth.
-
             time_step (float, optional):
                 The time step to be used by the model in :math:`s`.
 
-            planetary_radius (float, optional):
-                The radius of the planet to be used in :math:`m`. If None, the default
-                value from :code:`sympl.default_constants` is used.
-
-            planetary_rotation_rate (float, optional):
-                The rotation rate of the planet to be used in :math:`s^{-1}`. If None, the default
-                value from :code:`sympl.default_constants` is used.
-
-            universal_gas_constant (float):
-                value of the gas constant in :math:`J K^{-1} mol^{-1}`.
-                Default value from climt.default_constants is used if None.
-
-            gas_constant_dry_air (float, optional):
-                The gas constant for dry air in :math:`J kg^{-1} K^{-1}`.
-                If None, the default value in :code:`sympl.default_constants` is used.
-
-            gas_constant_condensible (float, optional):
-                The gas constant for the condensible substance in :math:`J kg^{-1} K^{-1}`.
-                If None, the default value in :code:`sympl.default_constants` is used.
-
-            acceleration_gravity (float):
-                value of acceleration due to gravity in
-                :math:`m s^{-1}`. If None, Default value from :code:`sympl.default_constants` is
-                used.
-
-           specific_heat_dry_air (float, optional):
-                The heat capacity of dry air in :math:`J kg^{-1} K^{-1}`.
-                If None, the default value in :code:`sympl.default_constants` is used.
-
-            specific_heat_condensible (float, optional):
-                The heat capacity of the condensible substance in :math:`J kg^{-1} K^{-1}`.
-                If None, the default value in :code:`sympl.default_constants` is used.
-
-
-
         """
-        if specific_heat_condensible is not None:
-            specific_heat_condensible = DataArray(
-                specific_heat_condensible, attrs={'units': 'J kg^-1 K^-1'})
 
         self._time_step = timedelta(seconds=time_step)
 
-        self._radius = replace_none_with_default(
-            'planetary_radius', planetary_radius)
+        self._radius = get_constant(
```

Comment:
In this file also, put these as single lines without a break where they fit on one line.